Package.swift

```diff
@@ -19,7 +19,10 @@ import PackageDescription
 var packageDependencies: [Package.Dependency] = [
   .package(url: "https://github.com/apple/swift-protobuf.git", .upToNextMinor(from: "1.1.1")),
   .package(url: "https://github.com/kylef/Commander.git", from: "0.8.0"),
-  .package(url: "https://github.com/apple/swift-nio-zlib-support.git", from: "1.0.0")
+  .package(url: "https://github.com/apple/swift-nio-zlib-support.git", from: "1.0.0"),
```

Comment:
We should version lock these more strictly I think. Maybe `upToNextMinor`?

Reply:
Done (except for `NIOHTTP2`).